hazelcast-management-center@5.X.rb

```diff
@@ -0,0 +1,26 @@
+class HazelcastManagementCenterAT5X < Formula
+    desc "Tool to run Hazelcast Management Center"
+    homepage "https://github.com/hazelcast/hazelcast-command-line"
```

Comment:
I think it's outdated, maybe something like 
```suggestion
    homepage "https://docs.hazelcast.com/management-center/5.0.4/getting-started/overview"
```